engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -81,6 +82,12 @@
       with a scope element that doesn't belong to the activating element's flow scope. \
       These variables should be set before or after the modification.""";
 
+  private static final String ERROR_MESSAGE_MORE_THAN_ONE_FLOW_SCOPE_INSTANCE =
+      """
+      Expected to modify instance of process '%s' but it contains one or more activate instructions \
+      for an element that has a flow scope with more than one active instances. Can't decide in \
+      which instance of the flow scope the element should be activated.""";
```

Comment:
🔧 I think we can make this message more descriptive and help users with what actions they should take to fix the problem. Maybe something like this:
```
Expected to modify instance of process '%s' but it contains one or more activate instructions for an element that has a flow scope with more than one active instance: '<activation element id>'. Please specify an ancestor element instance key for this activate instruction.
```

Reply:
Good point. I will adjust the message.

But I will not add the part with the ancestor (`Please specify an ancestor element instance key for this activate instruction.`) until the feature (#9646) is implemented.